design-toolkit/components/src/components/slider/index.tsx

```diff
@@ -286,42 +56,33 @@ export const Slider = ({
   className,
   showInput = false,
   showLabel = true,
-  layout = 'stacked',
+  layout = 'stack',
   value,
   defaultValue,
   label,
-  defaultRangeValue,
-  rangeValue,
   minValue = 0,
   maxValue = 100,
-  isRange = false,
   orientation = 'horizontal',
   ...props
 }: SliderProps) => {
-  const sliderValue = isRange ? rangeValue : value;
-  const sliderDefaultValue = isRange ? defaultRangeValue : defaultValue;
-
   return (
     <RACSlider
-      className={cn(sliderVariants({ orientation, layout }))}
+      className={slider()}
       minValue={minValue}
       maxValue={maxValue}
+      data-layout={layout}
       orientation={orientation}
-      value={sliderValue}
-      defaultValue={sliderDefaultValue}
+      value={value}
```

Comment:
Any props not being used elsewhere in the implementation should passthrough on `{...rest}` which should be at the top of the component